extensions/amp-story/1.0/amp-story.css

```diff
@@ -312,6 +312,30 @@ amp-story-page [data-text-background-color] {
   }
 }
 
+/* For amp-story-page-attachment-ui-v2 experiment. */
+.i-amphtml-story-attachment-active amp-story-page[active]::after {
```

Comment:
Is it possible to add this selector to the existing bookend overlay code, so we don't need to repeat the entire block?